coremltools/converters/mil/frontend/torch/utils.py

```diff
@@ -26,16 +26,22 @@
     1: torch.int8,
     2: torch.int16,
     3: torch.int32,
-    4: torch.int32,
```

Comment:
We should still keep this mapping in its original and true value
`4: torch.int64`

Reply:
Thanks for the feedback! Made the recommended changes as so 🤙 
```
NUM_TO_TORCH_DTYPE = {
    0: torch.uint8,
    1: torch.int8,
    2: torch.int16,
    3: torch.int32,
    4: torch.int64,
    5: torch.float16,
    6: torch.float32,
    7: torch.float64,
    11: torch.bool,
    12: torch.qint8,
    13: torch.quint8,
    14: torch.qint32,
```